rest/endpoints/playback_info/playback_info.yml

```diff
@@ -172,9 +172,55 @@ paths:
   /user_profiles/{user_profile_id}/assets/{asset_type}:
     get:
       description: |
-        * **Available since:** NOT YET AVAILABLE
+        * **Available since:** 2020-01 (BPLAT-12231)
       tags:
-        - Playback Info
-      summary: # TODO
-      operationId: indexPlaybackInfo
-      # TODO
+        - Playback Info Index
+      operationId: getPlaybackInfoIndex
+      parameters:
+        - in: path
+          name: user_profile_id
+          required: true
+          description: The identity of the user profile to get the playback info for.
+          schema:
+            type: string
+        - in: path
+          name: asset_type
+          required: true
+          description: The asset type of the playback info.
+          schema:
+            type: string
+            enum: ["recording", "catchup", "vod"]
+        - in: query
+          name: filter
+          description: Filter by group_id. Allowed filter ex. filter[groupId]='12345'
+          schema:
+            type: string
```

Comment:
Maybe write it so that it implies multiple filters are supported, but then specify only one supported filter, like how it's done for the program metadata API, i.e.
> Filter results, see [JSON API spec](http://jsonapi.org/recommendations/#filtering) for instructions. Supported attributes for filtering: `group_id`